source/features/vertical-front-matter.tsx

```diff
@@ -0,0 +1,43 @@
+import './vertical-front-matter.css';
+import React from 'dom-chef';
+import select from 'select-dom';
+import * as pageDetect from 'github-url-detection';
+
+import features from '.';
+
+function init(): void {
+	const table = select('[data-table-type="yaml-metadata"]');
+	if (!table) {
+		return;
+	}
+
```

Comment:
This can be skipped

Reply:
i guess there would be an error when `table` is `undefined` in `table.replaceWith(`

is an early return not always better to save time?

Reply:
> i guess there would be an error when `table` is `undefined` in `table.replaceWith(`

Indeed, I thought it was only used here, I didn't see that line